src/app/containers/SignupPage/index.js

```diff
@@ -33,6 +34,7 @@ function SignupPage({ history }) {
     officialContestant: null,
     officialStudent: null,
     iAgreeToTerms: null,
+    isPopup: 0,
```

Comment:
`isPopup` should not be put in `data`.

Move it out.

Reply:
```
const [showPopup, setShowPopup] = React.useState(false);
```

Reply:
If you need to use more values than `true` and `false`, don't just write `0`, `1`, `2` since they don't provide any context.

Add some global constants:
```
const POPUP_STATE = {
  NONE: 0;
  ERROR: 1;
  SUCCESS: 2;
}
```

```
const [popupState, setPopupState] = React.useState(POPUP_STATE.NONE);
```

```
if (!!error ...) {
  ...
  setPopupState(POPUP_STATE.ERROR)
  ...
}
```